salesforce/10.15/modules/ROOT/pages/index.adoc

```diff
@@ -49,6 +49,28 @@ Read xref:salesforce-connector-xml-maven.adoc[XML and Maven Support],
 the xref:salesforce-connector-config-topics.adoc[Additional Configuration Information] topic,
 and xref:salesforce-connector-examples.adoc[Examples].
 
+== Common Use Cases for the Connector
+
+Common use cases for Salesforce Connector include:
+
+* *ERP Integration*
++
+Give your sales team access to all relevant customer data from one system via real-time synchronization of data between Salesforce and your ERP & finance applications, for example, use Anypoint Connector for SAP (SAP Connector) to sync data between Salesforce and SAP and Anypoint Connector for NetSuite (NetSuite Connector) to sync data between Salesforce and NetSuite.
```

Comment:
Maybe make this 2 sentences? The second sentence could start at "For example..."